app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<header class="navbar">
+	<%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out-btn"%>
+	<h2 >Categories</h2>	
+</header>
+
+<div class="categories">
+	<ul  >
+		<% @groups.each do |category| %>
+		 <li class="category" >
+			  <%= image_tag category.icon, height: 50 , alt: 'icon'%> 
+			 <div class="category-info">
+				 <%= link_to category.name.capitalize, user_group_path(current_user, category)%>
+				 <p><span> $</span><%= category.purchases.sum { |item| item.entity.amount } %></p>
+			 </div>
+			 <%= button_to 'Delete', user_group_path(current_user, category), method: :delete, class: "delete-btn" %>
+		 </li>
+		<% end %>
+	</ul>
+	<div class="new-category" >
+		<%= link_to 'Add a new category', new_user_group_path(current_user) %>
+	</div>
```

Comment:
- [ ] Please adjust the styling of this button to look more like the original design.
- [ ] The same goes for the "Add a new transaction" button.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/235160400-53512556-6dab-4c12-b650-1fb616805d2a.png) | ![image](https://user-images.githubusercontent.com/62290545/235160463-21f97a59-2419-47de-a18c-c8893fed155c.png) |